latest/docs-ref-conceptual/tutorial-azure-cli-server.yml

```diff
@@ -0,0 +1,246 @@
+### YamlMime:YamlDocument
+documentType: Tutorial
+title: Deploy a web server in Azure
+metadata:
+  title: Deploy a web server in Azure
+  description: Get started with the Azure CLI by deploying a VM with the NGINX web server.
+  audience: IT Pro
+  level: Beginner
+  displayType: two-column
+  interactive: azurecli
+items:
+- durationInMinutes: 1
+  content: >
+    In this getting started tutorial for the Azure CLI, you will learn how to create a Linux virtual machine,
+    install the NGINX web server, configure a data disk to share between resources, and configure the virtual
+    network firewall to allow HTTP and HTTPS traffic. You will do all of these tasks through the Azure CLI,
+    which can be used in-browser as part of this tutorial. Or you can [install the Azure CLI](install-azure-cli.md)
+    and follow along in your own terminal.
+
+- title: Create a Virtual Machine
+  durationInMinutes: 4
+  content: |
+      First, log in with your Azure subscription information. Follow any on-screen commands to get
+      your shell configured and authenticate with Azure.
+
+      ```azurecli
+      az login
+      ```
+
+      The first step to creating resources with Azure is to have a resource group which will
+      contain and associate them. For this tutorial, we will want a dedicated resource group
```

Comment:
Create a dedicated resource group for this tutorial with the az group create command. This paragraph is too big for what it's doing. See other paragraphs in the Linux VMs area for more inspiraition.